interoperator/api/osb/v1alpha1/sfplan_types.go

```diff
@@ -78,6 +78,7 @@ type SFPlanSpec struct {
 	Free          bool                  `json:"free"`
 	Bindable      bool                  `json:"bindable"`
 	PlanUpdatable bool                  `json:"planUpdatable,omitempty"`
+	AutoUpdatable bool                  `json:"autoUpdatable,omitempty"`
```

Comment:
Its probably better to have the default behavior(when it is not set at all) as not doing autoUpdate. So the field name does not feel right. We should do auto update only if field is explicitly set to true.

Reply:
I am not sure about the naming, but you are right, it should kick in only when it is true. So if it is not set(which means it is false), it should not kick in. I have changed that logic. But, for the naming, let's discuss on that.